ql/src/java/org/apache/hadoop/hive/ql/udf/generic/GenericUDFSQCountCheck.java

```diff
@@ -26,22 +26,24 @@
 import org.apache.hadoop.hive.ql.exec.vector.expressions.gen.FuncAbsDoubleToDouble;
 import org.apache.hadoop.hive.ql.exec.vector.expressions.gen.FuncAbsLongToLong;
 import org.apache.hadoop.hive.ql.metadata.HiveException;
+import org.apache.hadoop.hive.ql.udf.UDFType;
 import org.apache.hadoop.hive.serde2.objectinspector.ObjectInspector;
 import org.apache.hadoop.hive.serde2.objectinspector.ObjectInspectorConverters;
 import org.apache.hadoop.hive.serde2.objectinspector.ObjectInspectorConverters.Converter;
 import org.apache.hadoop.hive.serde2.objectinspector.primitive.PrimitiveObjectInspectorFactory;
-import org.apache.hadoop.io.LongWritable;
+import org.apache.hadoop.io.BooleanWritable;
 
 /**
  * GenericUDFAbs.
  *
  */
+@UDFType(deterministic = false)
 @Description(name = "sq_count_check",
     value = "_FUNC_(x) - Internal check on scalar subquery expression to make sure atmost one row is returned",
     extended = "For internal use only")
 @VectorizedExpressions({FuncAbsLongToLong.class, FuncAbsDoubleToDouble.class, FuncAbsDecimalToDecimal.class})
```

Comment:
I don't think these Vectorized functions will work instead of this UDF

Reply:
This was fixed in HIVE-24934.